cmd/explaintest/main.go

```diff
@@ -40,13 +40,17 @@ import (
 const dbName = "test"
 
 var (
-	logLevel string
-	record   bool
-	create   bool
+	logLevel  string
+	port      uint
+	statsPort uint
```

Comment:
```suggestion
	statusPort uint
```
"stats" means "statistic".